src/templates/home-page/BTCRole/index.tsx

```diff
@@ -1,22 +1,29 @@
 import { FC } from "react"
+import { Divider, SimpleGrid } from "@chakra-ui/react"
 import SectionTemplate, { RoleTemplateProps } from "../SectionTemplate"
-import { TBTCPartnerImage, TBTCPartners } from "./tBTCPartners"
+import { renderPartners, TBTCPartner } from "../../../components/tBTCPartners"
+import { H4 } from "../../../components"
 
 interface BTCRoleProps {
-  minters: TBTCPartnerImage[]
-  guardians: TBTCPartnerImage[]
+  partners: TBTCPartner[]
 }
 
 export const BTCRole: FC<RoleTemplateProps & BTCRoleProps> = ({
-  minters,
-  guardians,
+  partners,
   ...props
-}) => {
-  return (
-    <>
-      <SectionTemplate {...props}>
-        <TBTCPartners minters={minters} guardians={guardians} mt={"6rem"} />
-      </SectionTemplate>
-    </>
-  )
-}
+}) => (
+  <>
+    <SectionTemplate {...props}>
+      <Divider my="5.5rem" />
+      <H4 color="gray.300">tBTC is backed by the following partners.</H4>
+      <SimpleGrid
+        mt="4.5rem"
+        columns={{ base: 2, md: 4, lg: 6 }}
+        spacingY="5.5rem"
+        justifyItems="center"
+      >
+        {partners.map(renderPartners)}
+      </SimpleGrid>
```

Comment:
Non-blocking:

We have a similar situation as above on the home page.

Before:
<img width="646" alt="image" src="https://github.com/threshold-network/splash-page/assets/40306834/ad9a71b0-67d6-412a-9e25-6fec016de785">

After:
<img width="819" alt="image" src="https://github.com/threshold-network/splash-page/assets/40306834/13c243ac-253e-44fa-81c2-b11755c4e18c">

Reply:
3189427